server/k8s/runner-manifest/runner.secrets.example.yaml

```
@@ -14,4 +14,6 @@ stringData:
   ACCESS_TOKEN_EXPIRATION_SEC: "300"
   OPENAI_API_KEY: very_secret
   CLIENT_URL: "http://localhost:3000"
-  GROQ_API_KEY: very_secret
\ No newline at end of file
+  GROQ_API_KEY: very_secret
+  OPENAI_API_KEY: very_secret
```

YAMLlint (1.37.1)

[error] duplication of key "OPENAI_API_KEY" in mapping

(key-duplicates)